stylesheets/highlight.scss

```diff
@@ -12,9 +12,7 @@ pre code {
 }
 
 pre .comment,
-pre .template_comment,
-pre .diff .header,
-pre .javadoc {
+pre .err {
```

Comment:
What do you think about scoping this rule so that it only affects JSON code blocks? Like `.language-json pre .err`